src/com/jambit/onboarding2020/tbrpg/domain/Room/BossRoom.java

```diff
@@ -53,7 +57,7 @@ private void fight(GameInput in, Enemy enemy, Random random) throws InvalidInput
         if (input == 1) {
             if (random.nextBoolean()) {
                 try {
-                    Player.getPlayerInstance().attack(enemy);
+                    Player.getPlayerInstance().attack(enemy, equippedWeapon);
```

Comment:
Man muss hier nicht equippedWeapon übergebn, da du einfach in der Methode ds Weapon abfragen kannst